src/SurvivalEstimator.jl

```diff
@@ -338,6 +338,128 @@ function StatsBase.fit!(obj::KaplanMeier, Y::RCSurv)
     )
 end
 
+#-------------------
+# Log-rank tests
+#-------------------
+function update_wt(wt, nevents, nrisk, wtmethod)
+    if wtmethod == :logrank
+        return 1
```

Comment:
```suggestion
        return 1.0
```

Reply:
@kshedden please merge this suggestion for better type consistency